### ISSUE TITLE
Haven't received my Bitconnect Token yet

### ISSUE THREAD

**Issue by mannygutierrez12** [bitconnectcoin/bitconnectcoin#64]
Hello community. I sent 28 bcc token into my bcc qt wallet. My wallet is working fine, it was not syncing when I sent the token, but I updated it and it is working perfect. The problem is I have not received my token yet. Is anybody having the same issues? How can I fix that. I have sent couple of email to the Bitconnect team, but they haven't responded yet.

**Comment by rickylaspinas22** [comment]
how to fix out of sync issues

**Comment by aloidflip** [comment]
What operating system are you using? Windows or Mac? I'm still having trouble syncing my wallet, I'm using windows 10 and same case I have sent my coins without having to sync it first. Keep me posted. THanks!

**Comment by ldspirit** [comment]
Yeah. I'm having the same issue here. Wallet has been synced and I made sure I backed it up before sending bcc token from the website. Still I have not received my coins and it happened 8 to 10 days ago!
